build.sbt

```diff
@@ -1393,6 +1394,14 @@ def archetypeProject(archetypeName: String) =
     )
 
 lazy val `maven-java-archetype` = archetypeProject("java")
+lazy val `bill-of-materials` = (project in file("dev") / "bill-of-materials")
+  .enablePlugins(SonatypeOnly, BillOfMaterialsPlugin)
+  .settings(sonatypeSettings, common, noMima, sbtScalaSettings, publishMavenStyleSettings)
+  .settings(
+    name := "lagom-bom",
+    bomIncludeProjects := coreProjects ++ javadslProjects ++ scaladslProjects,
+    pomExtra := pomExtra.value :+ bomDependenciesListing.value,
+  )
```

Comment:
It'd be good to have a [test](https://github.com/lagom/lagom/blob/master/dev/maven-plugin/src/maven-test/external-project-2.12/test) and docs for this new BOM.